gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/filter/JWTFederationFilter.java

```diff
@@ -75,41 +102,92 @@ public void destroy() {
   public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
       throws IOException, ServletException {
     String header = ((HttpServletRequest) request).getHeader("Authorization");
+    String header_hive = ((HttpServletRequest) request).getHeader("HiveAuthToken");
```

Comment:
Seems weird to have a special case for Hive here.

Reply:
WIP for it .. As beeline doesn't support space  like http.header.Authorization=Bearer eyJraWQiOi..     Thats why we used Hive specific header for  JWT ..